proto/catalog.proto

```diff
@@ -92,8 +92,9 @@ message Sink {
 message Connection {
```

Comment:
IIUC, the connection should belong to one of the databases and schemas and it should have one owner to fulfill the further privilege check in the future.

Reply:
I think the `database_id` and `schema_id` is needed for now in this PR. You can add a `TODO` here to support owner and further privilege check.

Reply:
currently in our implementation, the connection is "global" (a connection does not belong to a specific table/schema) I guess currently for this pr, we will get things working first, and then fix this in a later pr.
thanks a lot for your comments :)

Reply:
Okay, please add a `TODO` here.